### PR TITLE
[TypeScript][Skill]sampleDialog.ts v1.0 changes

### PR DESCRIPTION
**Pull request by matiasroldan6**
### Purpose
Add changes to sampleDialog.ts file to take to v1.0

### CODE REVIEW

File: templates/typescript/samples/sample-skill/src/dialogs/sampleDialog.ts [hunk 1]
```diff
@@ -63,7 +61,7 @@ export class SampleDialog extends SkillDialogBase {
         tokens.set(this.nameKey, stepContext.result as string);
```
Comment (Batta32) [inline templates/typescript/samples/sample-skill/src/dialogs/sampleDialog.ts:61]: Is this correctly migrated? I compared this to [C#](https://github.com/microsoft/botframework-solutions/blob/v1.0/samples/csharp/skill/SkillSample/Dialogs/SampleDialog.cs#L43) and the tokens is not used now.
Also, we can look the sampleAction -> [greetUser](https://github.com/southworks/botframework-solutions/blob/234cf5a5739eda6085dc0ea4d26a9a9e8ab0ecee/templates/typescript/samples/sample-skill/src/dialogs/sampleAction.ts#L66) method that you recently migrated.

File: templates/typescript/samples/sample-skill/src/dialogs/sampleDialog.ts [hunk 2]
```diff
@@ -63,7 +61,7 @@ export class SampleDialog extends SkillDialogBase {
...
 
         // eslint-disable-next-line @typescript-eslint/no-explicit-any
-        const response: any = this.templateManager.generateActivityForLocale('HaveNameMessage', stepContext.context.activity.locale, tokens);
+        const response: any = this.templateManager.generateActivityForLocale('HaveNameMessage', tokens, stepContext.context.activity.locale);
```
Comment (Batta32) [inline templates/typescript/samples/sample-skill/src/dialogs/sampleDialog.ts:64]: Following the comment of above, tokens should not be used

File: templates/typescript/samples/sample-skill/src/dialogs/sampleDialog.ts (hunk 1 above)
Comment (matiasroldan6) [reply templates/typescript/samples/sample-skill/src/dialogs/sampleDialog.ts:61]: C# never had a `tokens` variable, in between 0.8 or 1.0, but I understand this represents the variable `data` which is an object with a key-value: Name-stepContextResult. as string. You mean changing the name from `tokens `to `data`?

Comment (Batta32) [reply templates/typescript/samples/sample-skill/src/dialogs/sampleDialog.ts:61]: No, I mean that this method was not correctly migrated before, and it should be like the [greetUser](https://github.com/southworks/botframework-solutions/blob/234cf5a5739eda6085dc0ea4d26a9a9e8ab0ecee/templates/typescript/samples/sample-skill/src/dialogs/sampleAction.ts#L66) of `sampleAction`.

To understand this, compare both methods and identify that contains the same behavior/implementation in C#. However, in TypeScript differs.

**Pull request merged**